AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -143,7 +152,7 @@ def get_axon_morphometrics(im_axon, path_folder=None, im_myelin=None, pixel_size
                 stats['myelin_thickness'] = myelin_thickness
                 stats['myelin_area'] = myelin_area
                 stats['axonmyelin_area'] = axonmyelin_area
-                stats['gratio'] = np.sqrt(axon_area / axonmyelin_area)
+                stats['gratio'] = (axon_diam / 2) / (axon_diam / 2 + myelin_thickness)
```

Comment:
I realized that in the last meeting I wasn't able to properly put my point forward of why I went on to change the gratio from `np.sqrt(axon_area / axonmyelin_area)` to `(axon_diam / 2) / (axon_diam / 2 + myelin_thickness)`. Well, gratio is defined as `axon_radius / (axon_radius + myelin thickness)`, which is the general formula that would fit in for the both **ellipse** as well as **circle** shaped axons.  

If we set the gratio as `np.sqrt(axon_area / axonmyelin_area)`, this wouldn't work for the case when the axons are ellipsoidal in shape and the gratio measured in this case would be inaccurate because it would be similar to that when axons are circular in shape ( as the area of the axons and the axon + myelin would remain same for the ellipsoidal shape)

Hence, I decided to drop off the earlier formula of gratio which was dependent on the area of the axon and axon + myelin.